README.md

```diff
@@ -20,30 +20,54 @@ composer require utopia-php/http
 
 Init your first application in `src/server.php`:
 
+
 ```php
 require_once __DIR__.'/../vendor/autoload.php';
 
+use Utopia\DI\Container;
+use Utopia\DI\Dependency;
 use Utopia\Http\Http;
 use Utopia\Http\Request;
 use Utopia\Http\Response;
 use Utopia\Http\Adapter\FPM\Server;
 
-Http::get('/hello-world') // Define Route
-    ->inject('request')
-    ->inject('response')
+class User {
+    public string $name;
+    
+    public function __construct(string $name)
+    {
+        $this->name = $name;
+    }
+}
+// Creating the dependency injection container
+$container = new Container();
+
+// Adding a user dependency to the container
+$user = new Dependency();
+$user
+    ->setName('user')
```

Comment:
I think it would be cool to include dependency inside resource example. we could inject request and use x-user-id to get user ID